centreon/tests/php/App/ActivityLogging/Infrastructure/Double/FakeActivityLogFactory.php

```
@@ -0,0 +1,58 @@
+use App\ActivityLogging\Domain\Aggregate\TargetId;
+use App\ActivityLogging\Domain\Aggregate\TargetName;
+use App\ActivityLogging\Domain\Aggregate\TargetTypeEnum;
+use App\ActivityLogging\Domain\Factory\ActivityLogFactoryInterface;
+use App\Shared\Domain\Aggregate\AggregateRoot;
+
+/**
+ * @implements ActivityLogFactoryInterface<AggregateRoot>
+ */
+final class FakeActivityLogFactory implements ActivityLogFactoryInterface
+{
+    public function create(ActionEnum $action, AggregateRoot $aggregate, Actor $firedBy, \DateTimeImmutable $firedAt): ActivityLog
```

PHPMD (2.15.0)

Avoid unused parameters such as '$aggregate'. (Unused Code Rules)

(UnusedFormalParameter)